---
title: "An alternative route of bacterial infection is associated with a polymorphism at an alternative resistance locus."

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors: 
-Bento G
-Fields P
-admin
-Ebert D

# Author notes (optional)
#author_notes:
#- "Equal contribution"
#- "Equal contribution"

date: "2018-12-01T00:00:00Z"
doi: "10.1038/s41437-020-0332-x"

# Schedule page publish date (NOT publication's date).
publishDate: "2017-12-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: In *Heredity*

abstract: To understand the mechanisms of antagonistic coevolution, it is crucial to identify the genetics of parasite resistance. In the Daphnia magna–Pasteuria ramosa host–parasite system, the most important step of the infection process is the one in which P. ramosa spores attach to the host’s foregut. A matching-allele model (MAM) describes the host–parasite genetic interactions underlying attachment success. Here we describe a new P. ramosa genotype, P15, which, unlike previously studied genotypes, attaches to the host’s hindgut, not to its foregut. Host resistance to P15 attachment shows great diversity across natural populations. In contrast to P. ramosa genotypes that use foregut attachment, P15 shows some quantitative variation in attachment success and does not always lead to successful infections, suggesting that hindgut attachment represents a less-efficient infection mechanism than foregut attachment. Using a Quantitative Trait Locus (QTL) approach, we detect two significant QTLs in the host genome: one that co-localizes with the previously described D. magna PR locus of resistance to foregut attachment, and a second, major QTL located in an unlinked genomic region. We find no evidence of epistasis. Fine mapping reveals a genomic region, the D locus, of ~13 kb. The discovery of a second P. ramosa attachment site and of a novel host-resistance locus increases the complexity of this system, with implications for both for the coevolutionary dynamics (e.g., Red Queen and the role of recombination), and for the evolution and epidemiology of the infection process.

# Summary. An optional shortened abstract.
summary: To understand the mechanisms of antagonistic coevolution, it is crucial to identify the genetics of parasite resistance. Using QTL approach, we discovered a second P. ramosa attachment site and a novel host-resistance locus, with implications for both for the coevolutionary dynamics (e.g., Red Queen and the role of recombination), and for the evolution and epidemiology of the infection process.

tags: []

# Display this page in the Featured widget?
featured: false

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: 'publication/Manuscript/Bento_Heredity_2020.pdf'
#url_code: ''
#url_dataset: ''
#url_poster: ''
#url_project: ''
#url_slides: ''
#url_source: ''
#url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**David Duneau**]'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
- Genetics_basis

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
#slides: example
---

blabla
